src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -33,12 +38,21 @@ class Meta:
         }
 
 
+@extend_schema_serializer(deprecate_fields=["toegewezen_aan_actor"])
 class InterneTaakSerializer(serializers.HyperlinkedModelSerializer):
     toegewezen_aan_actor = ActorForeignKeySerializer(
-        required=True,
+        required=False,
```

Comment:
Now it's possible that a task doesn't have any actor, isn't it? 
Is this change intended? Can it affect a process in some way?

Reply:
It is indeed not intended for a `IntereTaak` to not have an `Actor`.
But because we now have 2 (1 temporary) `Actor` fields we can't have them set as required.
So I moved the "required" clause to the validation function within the serializer.